src/Config/user_local.props

```diff
@@ -8,7 +8,7 @@
     <TestOutputPath Condition=" '$(TestOutputPath)' == '' ">$(OutputPath)</TestOutputPath>
   </PropertyGroup>
   <PropertyGroup>
-    <REVITAPI Condition=" !Exists('$(REVITAPI)') ">$(SolutionDir)..\lib\Revit 2025\net8.0</REVITAPI>
+    <REVITAPI Condition=" !Exists('$(REVITAPI)') ">C:\Program Files\Autodesk\Revit 2025</REVITAPI>
```

Comment:
Also, in the future can you, please, add an entry instead of replacing this one? The current path is used by the internal build pipeline to download Revit API. Just adding another with the current condition `!Exist('$(REVITAPI)')` should work just as well.